validator/entrypoint.sh

```diff
@@ -6,32 +6,34 @@ WEB3SIGNER_API="http://web3signer.web3signer-${NETWORK}.dappnode:9000"
 
 # MEVBOOST: https://docs.teku.consensys.net/en/latest/HowTo/Builder-Network/
 if [ -n "$_DAPPNODE_GLOBAL_MEVBOOST_PRATER" ] && [ "$_DAPPNODE_GLOBAL_MEVBOOST_PRATER" == "true" ]; then
-  echo "MEVBOOST is enabled"
+  echo "MEV-BOOST is Enabled"
   MEVBOOST_URL="http://mev-boost.mev-boost-goerli.dappnode:18550"
-  if curl --retry 5 --retry-delay 5 --retry-all-errors "${MEVBOOST_URL}"; then
-    EXTRA_OPTS="--validators-builder-registration-default-enabled ${EXTRA_OPTS}"
-  else
-    echo "MEVBOOST is enabled but ${MEVBOOST_URL} is not reachable"
-    curl -X POST -G 'http://my.dappnode/notification-send' --data-urlencode 'type=danger' --data-urlencode title="${MEVBOOST_URL} is not available" --data-urlencode 'body=Make sure the mevboost is available and running'
+  EXTRA_OPTS="--validators-builder-registration-default-enabled --validators-proposer-blinded-blocks-enabled=true ${EXTRA_OPTS}"
+  if curl --retry 5 --retry-delay 5 --retry-connrefused "${MEVBOOST_URL}"; then
```

Comment:
I think this is wrong @alexpeterson91 . If the curl GET is successful, then MEV-BOOST is reachable